xcell/cls/data.py

```diff
@@ -284,6 +284,9 @@ def get_mapper(self, tr):
         config = self.data['tracers'][tr]
         nside = self.data['sphere']['nside']
         coords = self.data['sphere']['coords']
+        config['ignore_rerun'] = config.get('ignore_rerun',
+                                            self.data.get('ignore_rerun',
+                                                          False))
```

Comment:
Hmm, is this a global variable? I think it could also be helpful to have it defined per tracer (imagine that you change a mapper which modifies the mask you will only want to ignore the rerun for that case). Also, will `ignore_rerun` overwrite what you have?

Reply:
This allows you to have it defined per tracer, and if it is, then that overrides the global variable.

Reply:
Right. My brain ignored the `config.get` part...